src/index.ts

```diff
@@ -134,3 +134,6 @@ export { ProcessListItem } from './components/ProcessList/ProcessListItem/Proces
 export { ProcessListHeading } from './components/ProcessList/ProcessListHeading/ProcessListHeading'
 
 export { SiteAlert } from './components/SiteAlert/SiteAlert'
+
+/** Types */
+export type { HeadingLevel } from './types/headingLevel'
```

Comment:
Rather than forcing prop structs like `items` on `Accordion` to use `const items: ComponentProps<typeof Accordion>['items'] = {...}`, which is a bit gross, this export was added to allow the `headingLevel` prop to be declared as `headingLevel: "h4" as HeadingLevel` if consumers want to go that route.